change/react-native-windows-2020-05-06-15-51-59-autolinking.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Autolinking",
```

Comment:
FYI this comment shows up in change logs. Right now we have one for 0.62 that says it has autolinking 😉